packages/db/src/async_operations.ts

```diff
@@ -0,0 +1,19 @@
+import { toSnake } from 'snake-camel';
+import { ApiAsyncOperation } from '@cumulus/types/api/async_operations';
+import { PostgresAsyncOperation } from './types';
+
+export const translateApiAsyncOperationToPostgresAsyncOperation = (
+  record: ApiAsyncOperation
+): PostgresAsyncOperation => {
+  const translatedRecord: PostgresAsyncOperation = <PostgresAsyncOperation>toSnake(record);
```

Comment:
I could be wrong, but I don't know if we need `PostgresAsyncOperation` on both sides of the assignment